libs/designsystem/src/lib/components/list/list.component.scss

```diff
@@ -160,10 +159,29 @@ ion-item-options[side='end'] {
     border-radius: $border-radius;
   }
 
+  .virtual-with-sections-wrapper {
+    margin-left: -1 * size('xxxs');
+    margin-right: -1 * size('xxxs');
+  }
+
   .virtual-group {
     margin-top: size('m');
   }
 
+  .virtual-section-item {
+    box-shadow: $shadow;
+    margin: 0 size('xxxs');
```

Comment:
This margin causes the virtual scrolling list with sections to be of less width than an ordinary list.